src/ape/api/explorers.py

```diff
@@ -52,7 +54,7 @@ def get_contract_type(self, address: AddressType) -> Optional[ContractType]:
             address (:class:`~ape.types.AddressType`): The contract address.
 
         Returns:
-            Optional[:class:`~ape.contracts.ContractType`]: If not published, returns ``None``.
+            Optional[:class:`ethpm_types.contract_type.ContractType` if published, else ``None``.
```

Comment:
I don't know if we can put a whole sentence in the returns type docs, can we? I thought it had to be like a single word and then followed by a colon.